### ISSUE TITLE
dependant collection seeding not working for collection other than `Meteor.users`

### ISSUE THREAD

**Issue by dtamburr** [cleverbeagle/seeder#4]
When trying to seed dependant collections, it seems that the `data` function field is never called, unless the collection is `Meteor.users`.

Let's take the code from `pup` and add a `Videos` collection. The `Videos` collection is seeded, but not the `Documents`

```js
import Videos from '../../api/Videos/Videos'; // Videos collection

const documentsSeed = ...

seeder(Videos, {
  environments: ['development', 'staging'],
  wipe: true,
  noLimit: true,
  modelCount: 3,
  model(index) {
    console.log(`creating videos ${index + 1}`);
    return {
      title: `Title of video #${index + 1}`,
      data(videoId) {
        console.log('This is never called');
        return documentSeed(videoId);
      },
    };
  },
});
```

**Comment by pilarArr** [comment]
I have this issue too.

**Comment by kemo71** [comment]
Me too

**Comment by rangiesrule** [comment]
Same for me.

Thought it was something I was doing. Trying to create a related collection of the Documents collection adding the data method to the return of the documentsSeed method.

**Comment by rangiesrule** [comment]
I have been able to seed another related collection by getting list of _id's after seeder has finished and iterated over the list with a seeder method populating the collection.

Not as clean if the data method did the trick.

**Comment by pilarArr** [comment]
@rangiesrule how did you get the list of ids?

**Comment by rangiesrule** [comment]
After the first seed, created an array of _ids with a find on the new collection. Then placed the related seeder method inside a for loop to seed the related items with the parent id.

**Comment by SergeJScr** [comment]
Solution here
https://github.com/cleverbeagle/seeder/issues/7

**Comment by cleverbeagle** [comment]
Fixed and available in 1.3.0 :)

**Comment by SergeJScr** [comment]
Cool!
